silkworm/silkrpc/core/account_dumper.cpp

```diff
@@ -115,7 +115,7 @@ Task<void> AccountDumper::load_accounts(ethdb::TransactionDatabase& tx_database,
     co_return;
 }
 
-Task<void> AccountDumper::load_storage(uint64_t block_number, DumpAccounts& dump_accounts) {
+Task<void> AccountDumper::load_storage(BlockNum block_number, DumpAccounts& dump_accounts) {
```

Comment:
The same change should be applied also in `account_dumper.hpp`

Reply:
ok